website/content/en/docs/deployment/cognito/guide.md

```diff
@@ -60,7 +60,7 @@ From this point onwards, we will be creating/updating the DNS records **only in
 
 ## 3.0 Configure Ingress
 
-1. Take note of the following values from the previous step or `awsconfigs/infra_configs/scripts/config.yaml` if you used the [automated guide](https://github.com/awslabs/kubeflow-manifests/blob/main/docs/deployment/cognito/README-automated.md):
+1. Take note of the following values from the previous step or `awsconfigs/infra_configs/scripts/config.yaml` if you used the [automated guide](https://github.com/awslabs/kubeflow-manifests/blob/main/deployments/cognito/README-automated.md):
```

Comment:
change this to: Take note of the following values from the previous step

drop the rest. Its a bug we missed previously